kasa/exceptions.py

```diff
@@ -11,3 +12,87 @@ class UnsupportedDeviceException(SmartDeviceException):
 
 class AuthenticationException(SmartDeviceException):
     """Base exception for device authentication errors."""
+
+
+class RetryableException(SmartDeviceException):
+    """Retryable exception for device errors."""
+
+
+class TimeoutException(SmartDeviceException):
+    """Timeout exception for device errors."""
+
+
+class SmartErrorCode(Enum):
```

Comment:
```suggestion
class SmartErrorCode(IntEnum):
```
This allows comparing them like they were ints.

Reply:
Done